ts/components/MemberListItem.tsx

```diff
@@ -86,25 +116,204 @@ export const MemberListItem = (props: {
   isAdmin?: boolean; // if true,  we add a small crown on top of their avatar
   onSelect?: (pubkey: string) => void;
   onUnselect?: (pubkey: string) => void;
+  dataTestId?: React.SessionDataTestId;
+  displayGroupStatus?: boolean;
+  groupPk?: string;
   disabled?: boolean;
-  dataTestId?: string;
-}) => {
-  const {
-    isSelected,
-    pubkey,
-    isZombie,
-    isAdmin,
-    onSelect,
-    onUnselect,
-    inMentions,
-    disableBg,
-    withBorder = true,
-    maxNameWidth,
-    disabled,
-    dataTestId,
-  } = props;
+  hideRadioButton?: boolean;
+};
+
+const ResendContainer = ({
+  displayGroupStatus,
+  groupPk,
+  pubkey,
+}: Pick<MemberListItemProps, 'displayGroupStatus' | 'pubkey' | 'groupPk'>) => {
+  const weAreAdmin = useWeAreAdmin(groupPk);
+
+  if (
+    weAreAdmin &&
+    displayGroupStatus &&
+    groupPk &&
+    PubKey.is03Pubkey(groupPk) &&
+    PubKey.is05Pubkey(pubkey) &&
+    !UserUtils.isUsFromCache(pubkey)
+  ) {
+    return (
+      <Flex
+        container={true}
+        margin="0 0 0 auto"
+        padding="0 var(--margins-lg)"
+        gap="var(--margins-sm)"
+      >
+        <ResendButton groupPk={groupPk} pubkey={pubkey} />
+        <PromoteButton groupPk={groupPk} pubkey={pubkey} />
+      </Flex>
+    );
+  }
+  return null;
+};
+
+const StyledGroupStatusText = styled.span<{ isFailure: boolean }>`
+  color: ${props => (props.isFailure ? 'var(--danger-color)' : 'var(--text-secondary-color)')};
+  font-size: var(--font-size-xs);
+  margin-top: var(--margins-xs);
+  min-width: 100px; // min-width so that the dialog does not resize when the status change to sending
+  text-align: left;
```

Comment:
Is this specifically left or should it be text-align: start instead?

Reply:
that's right, it should be :start. Updated